### ISSUE TITLE
Getting `Error: RNEncryptedStorageError: An error occured while removing value` on sign out

### ISSUE THREAD

**Issue by nmalancea** [iteratehq/react-native-iterate#75]
I'm getting the following error when signing out of the app even though I call `Iterate.reset()` as recommended in the docs.
```
Possible Unhandled Promise Rejection (id: 9):
Error: RNEncryptedStorageError: An error occured while removing value
promiseMethodWrapper@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false:3511:45
removeItem@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false:253883:45
```

Is this a known issue?

I get this error twice on each sign out.

**Comment by msingleton** [comment]
@nmalancea thanks for the report, we'll take a look and get back to you shortly

**Comment by msingleton** [comment]
@nmalancea this is fixed in version 2.2.1 which was just published

**Comment by nmalancea** [comment]
Awesome! Thank you @msingleton. The error is gone.
